### PR TITLE
Add active activity border and background colors

### LINKED ISSUE #75435: [Feature] Highlight current activity in activity bar more explicitly
I am partially color blind and I have trouble differentiating between the shades of color in the activity bar.

For example, in my current workflow I have the activity bar layout as follows: 
VS Code Current Tab Highlight
Might be clear to some of you that it is the icon after the *Source Control* activity (Git Lens) but I cannot tell without serious concentration.

It would be nice to see a more explicit highlight of the current activity.

For example, here is a screen shot from Microsoft Teams:
Teams Current Tab Highlight

Suggested labels: `feature-request`, `accessibility`

Thank you very much for your time and effort. I am deeply grateful.

### PR DESCRIPTION
**Pull request by miguelsolorio**
Fixes #75435

This tweaks the opacity of `activityBar.inactiveForeground` to make the active element stand out more, and also adds two new color tokens:

- `activityBar.activeBorder` which is on by default
- `activityBar.activeBackground` which is off by default

gif

_Note: I hide the active border on high contrast mode as we already have an active indicator. Otherwise it re-uses the same styles/pattern for high contrast._

This follows the same convention as `tab.activeBorder` and when configured can look like this:

image

@aeschli for color token names + how it handles the colors in HC mode

### CODE REVIEW

File: src/vs/workbench/browser/parts/compositeBarActions.ts [hunk 1]
```diff
@@ -211,6 +213,9 @@ export class ActivityActionViewItem extends BaseActionViewItem {
 		this.badge = dom.append(container, dom.$('.badge'));
 		this.badgeContent = dom.append(this.badge, dom.$('.badge-content'));
 
+		// Active border + background
+		dom.append(container, dom.$('.active-item'));
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/compositeBarActions.ts]: @misolori why is this needed? Note that the composite switcher is being used both in activity bar and panel area.

**Review by bpasero — CHANGES_REQUESTED** [review]
I feel we now have a inconsistent default look compared to the panel area where we do not use a white border but gray:
image

Glitch: when both border and background is set, border does not seem to be on the far left:
image

File: src/vs/workbench/browser/parts/compositeBarActions.ts (hunk 1 above)
Comment (misolori) [reply src/vs/workbench/browser/parts/compositeBarActions.ts]: In order for us to set a border + bg color, it needs a dedicated dom element as the activity bar already uses the `:before` element and bg color for activity bar foreground color. I can see if I can hide this when not in use on the activity bar.

**Comment by misolori** [comment]
> I feel we now have a inconsistent default look compared to the panel area where we do not use a white border but gray

The panel's active foreground is different than the activity bar, but I'm already planning on making those consistent in #68453 (see https://github.com/microsoft/vscode/issues/68453#issuecomment-539622741)

> Glitch: when both border and background is set, border does not seem to be on the far left:

I used the same styling from the `focus` element and it appears that they move it by a pixel. Will see if I can clean this up.

File: src/vs/workbench/browser/parts/compositeBarActions.ts (hunk 1 above)
Comment (misolori) [reply src/vs/workbench/browser/parts/compositeBarActions.ts]: Wrapped this in an if statement so it's only shown on the activity bar

Comment (misolori) [reply src/vs/workbench/browser/parts/compositeBarActions.ts]: Wrapped this in an if statement so it's only shown on the activity bar

**Comment by misolori** [comment]
@bpasero updated this to address your feedback, borders are now accurate and match our other tabs + borders:

image

**Review by bpasero — APPROVED** [review]
